tests/System.Buffers.Primitives.Tests/ReadableBufferFacts.cs

```diff
@@ -180,5 +181,129 @@ public void Create_WorksWithOwnedMemory()
             b => b.Slice(0, 70).Slice(0, b.End),
             b => b.Slice(70, b.Start)
         };
+
+        static bool TryGetBuffer(Position begin, Position end, out ReadOnlyMemory<byte> data, out Position next)
```

Comment:
Why is this here?

Reply:
I can move the tests that call Seek to Pipelines.Tests and then eliminate it. What do you think?

Reply:
These tests were testing internal implementations of `ROB` why would we want to move them to `Pipelines`? Or even worse duplicate code being tested inside the test itself.

Reply:
What do you propose I do? Is there a way to test this without calling ROB's private APIs?

Reply:
Make them internal. It has been done in other places to verify implementation details.